appveyor.yml

```diff
@@ -0,0 +1,164 @@
+---
+
+# borrowed heavily from ogrisel/python-appveyor-demo
+
+version: "0.0.1.{build}"
+
+environment:
+  matrix:
+    ##################################################
+    # NOTE: For now, Fortran features are not tested #
+    ##################################################
+
+    # Visual Studio (Python 2 & 3, 32 & 64 bit)
+    - PYTHON: "C:\\Python27"
+      PYTHON_VERSION: "2.7.x"
+      PYTHON_ARCH: "32"
+      CMAKE_GENERATOR: Visual Studio 9 2008
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run-with-visual-studio.cmd"
+      EXTRA_TEST_ARGS: "--exclude fortran"
```

Comment:
Should this be `--exclude fortran_compiler` ?

See https://github.com/opadron/scikit-build/blob/169667a30776de70cb3529880afdd5ba53c0530f/tests/test_platform.py#L52